tests/PHPStan/Command/CaptureOutput.php

```diff
@@ -0,0 +1,36 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Command;
+
+class CaptureOutput implements Output
+{
+
+	/** @var string */
+	private $result = '';
+
+	public function writeFormatted(string $message): void
+	{
+		throw new \Exception('Not implemented');
```

Comment:
Class that implements an interface but throws exceptions in some of the methods breaks Liskov substitution principle. Please implement those methods even if they're unused.

Reply:
Changed this and created an anonymous class for ->getStyle() return.